crates/iceberg-catalog/src/catalog/tables.rs

```diff
@@ -929,3 +965,65 @@ fn maybe_body_to_json(request: impl Serialize) -> serde_json::Value {
         serde_json::Value::Null
     }
 }
+
+// TableUpdate is not clone but all containing fields are so we use this function to clone..
+fn copy_updates(updates: &[TableUpdate]) -> Vec<TableUpdate> {
```

Comment:
Should we maybe rename to clone_updates?

https://github.com/apache/iceberg-rust/pull/402

Reply:
done